packages/integrations/markdoc/package.json

```diff
@@ -59,7 +59,7 @@
     "build": "astro-scripts build \"src/**/*.ts\" && tsc",
     "build:ci": "astro-scripts build \"src/**/*.ts\"",
     "dev": "astro-scripts dev \"src/**/*.ts\"",
-    "test": "astro-scripts test --timeout 40000 \"test/**/*.test.js\""
+    "test": "astro-scripts test --timeout 40000 --parallel \"test/**/*.test.js\""
```

Comment:
Reminder that we should remove this before merging the PR

Reply:
I'm still unsure how to resolve my testing issue without this adjustment. Will try to discuss in team-platform

Reply:
Does that mean that without `--parallel` your test will always fail?